packages/palette/tslint.json

```diff
@@ -1,38 +0,0 @@
-{
-  "extends": ["tslint:recommended", "tslint-react", "tslint-config-prettier"],
-  "rules": {
-    "curly": false,
-    "interface-name": [true, "never-prefix"],
-    "jsx-boolean-value": [true, "never"],
-    "jsx-no-lambda": false,
-    "jsx-no-bind": false,
-    "max-classes-per-file": false,
-    "member-access": [false],
-    "member-ordering": false,
-    "no-unused-expression": false,
-    "no-var-requires": false,
-    "object-literal-sort-keys": false,
-    "ordered-imports": true,
-    "no-console": [true, "log"],
-    "no-namespace": [true, "allow-declarations"],
-    "trailing-comma": false,
-    "variable-name": [true, "ban-keywords", "allow-leading-underscore"],
-    "completed-docs": {
-      "severity": "warning",
-      "options": {
-        "functions": {
-          "visibilities": ["exported"]
-        },
-        "classes": {
-          "visibilities": ["exported"]
-        },
-        "types": {
-          "visibilities": ["exported"]
-        },
-        "variables": {
-          "visibilities": ["exported"]
-        }
-      }
-    }
-  }
-}
```

Comment:
So I didn't migrate all of these obviously. I tried using a tool to do it automatically but it requires us to update tslint to even use it.

If there's anything that stands out as being critical here lemme know?

Reply:
Nah, it's all good. (Those migration tools are crazy)